src/allocators.h

```diff
@@ -124,7 +123,7 @@ static inline size_t GetSystemPageSize()
     page_size = sSysInfo.dwPageSize;
 #elif defined(PAGESIZE) // defined in limits.h
     page_size = PAGESIZE;
-#else // assume some POSIX OS
+#else                   // assume some POSIX OS
```

Comment:
That's where we disagree, excessive white space is never good in my opinion. Personally I'm not for aligning trailing comments, there's literally no reason for it. Trailing comments are almost never in any way related to each other.